.travis/deploy.sh

```diff
@@ -16,8 +16,23 @@ if [[ "$TRAVIS_OS_NAME" == "linux" && ${BUILD_LINUX_WHEELS} -eq 1 ]]; then
     .travis/build_windows_wheels.sh
 else
 	if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-		python -m pip install wheel
-		python setup.py bdist_wheel
+		# Make sure we can build and "fix" the wheel.
+		python -m pip install delocate wheel
+		# Create directories for the built and fixed wheels.
+		mkdir dist_wheels/ fixed_wheels/
+		# Build the wheel for the local OS.
+		python -m pip wheel . --wheel-dir dist_wheels/
```

Comment:
@ofek I made the assumption that `deploy.sh` is run from the project root. If not, then `pip wheel .` will need to be changed to refer to a different path.

Also, you may be out of luck using `delocate` for PyPy on OS X. It might work, I'm not sure. I'm also not sure what `auditwheel` does on OS X.